Makefile

```diff
@@ -291,10 +291,11 @@ $(LOCALBIN):
 ##@ Tooling Binaries:
 # setup-envtest does not have devbox support so always use CACHE_BIN
 
-KUBECTL        ?= kubectl
+KUBECTL        ?= $(LOCALBIN)/kubectl
```

Comment:
will this work outside of devbox without a kubectl target?

Reply:
Do you mean `kubebuilder`? Yes, it should. 

Actually, I don't even think we have a `kubectl` target in the Makefile right now so this is just a floating variable.

Reply:
yeah, specifically the kubectl binary since before it was just using the host kubectl wherever that's installed vs installing and using whatever kubectl version in the tool path

Reply:
Oh I meant like we only have this variable definition in the Makefile to define the local `kubectl` binary path, and not any other rules/targets that do anything with this variable. 

The `kubectl` binary used should be whatever is in your `$PATH`.